### PR TITLE
replace some string concatenation (small clean up)

### PR DESCRIPTION
**Pull request by AxeZhan**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Replace some simple string concatenation to a function `GetNamespacedName`.
I think this is more clear and less error prone.
Besides, I think we should unify the usage:)
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/volumebinding/binder.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package volumebinding
 import (
 	"context"
 	"fmt"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
```
Comment (sanposhiho) [inline pkg/scheduler/framework/plugins/volumebinding/binder.go:22]: Please move this import to the below section. (L26-L48)

**Review by sanposhiho — COMMENTED** [review]
/lgtm

File: pkg/scheduler/framework/plugins/volumebinding/binder.go [hunk 2]
```diff
@@ -461,11 +462,11 @@ func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod, podVolumes *PodVolumes
 }
 
 func getPodName(pod *v1.Pod) string {
-	return pod.Namespace + "/" + pod.Name
+	return framework.GetNamespacedName(pod.Namespace, pod.Name)
```
Comment (denkensk) [inline pkg/scheduler/framework/plugins/volumebinding/binder.go:465]: It's better to just replace `getPodName` and `getPVCName` with `GetNamespacedName` directly.

Comment (kidddddddddddddddddddddd) [reply pkg/scheduler/framework/plugins/volumebinding/binder.go:465]: Makes sense.I'll do a quick replace

**Comment by dims** [comment]
If you still need this PR then please rebase, if not, please close the PR

**Pull request closed without merging**